sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/TransactionalBatchItemRequestOptionsBase.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.RequestOptions;
+import com.azure.cosmos.util.Beta;
+
+/**
+ * Encapsulates options that can be specified for an operation within a {@link TransactionalBatch}.
+ */
+@Beta(value = Beta.SinceVersion.V4_7_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
```

Comment:
4.13 - xxBase didn't exist before - only the derived class

Reply:
I thought I was restructuring the code so left the version as is.